docs/hello.md

```diff
@@ -0,0 +1,101 @@
+---
+id: hello
+title: Hello World
+sidebar_label: Hello World
+---
+
+This guide walks you through through mocking a simple service. Our service will be called `hello`, and it describes a request made to `api.unmock.io`. The root endpoint returns a single value: `hello` - which is a string.
+
+Note: Unmock documentation uses yarn, but npm will also work. You can compare yarn and npm [here](https://yarnpkg.com/en/docs/migrating-from-npm#toc-cli-commands-comparison).
+
+## Install unmock, jest, and axios
+
+First, let's create a project called `hello-unmock`.
+
+```bash
+$ mkdir hello-unmock && cd hello-unmock
+$ yarn init -y
```

Comment:
Generally, as we're swapping some stuff out - do we want to use `yarn` throughout the examples or `npm`? :thinking:

Reply:
I think the Jest way of using `yarn` and attaching "Note: Jest documentation uses yarn commands, but npm will also work. You can compare yarn and npm commands in the yarn docs, here." somewhere in the docs works well.